suite-native/atoms/src/Input/Input.tsx

```diff
@@ -78,14 +78,18 @@ const inputStyle = prepareNativeStyle(utils => ({
 
 const inputLabelStyle = prepareNativeStyle(
     (utils, { isLabelMinimized }: Pick<InputWrapperStyleProps, 'isLabelMinimized'>) => ({
-        ...utils.typography.body,
+        lineHeight: utils.typography.body.lineHeight,
+        letterSpacing: utils.typography.body.letterSpacing,
```

Comment:
I kind of dislike that we have to write-out everything manually, because it leaves space for human error, I would probably prefer approach using [`deleteKey`](https://mobily.github.io/ts-belt/api/object#deletekey)

`...D.deleteKey(utils.typography.body, 'fontSize')`

Reply:
fontSize should be handled by the animation style itself, otherwise, it always blinks with fontSize utils.typography.sth

Reply:
https://github.com/trezor/trezor-suite/pull/5700/commits/77511b5f42336b8195307300c93f5d50570f267a